app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -47,65 +47,80 @@ import {
 } from '../../common/analytics/analytics-events';
 import { AnalyticsClient } from '../../common/analytics/analytics-client';
 import { ParamTypes } from '../../common/types';
+import { CopiedToClipboard } from '../CopiedToClipboard/CopiedToClipboard';
 
 const analyticsClient = new AnalyticsClient();
 
 type CopyProps = {
 	handleCopyToClipboard: (copyRef: React.RefObject<HTMLDivElement>, elementName?: string) => Promise<void> | void;
+	setCopyButtonStateToTrue: (copyButton: CopyButtonNameEnum) => void;
```

Comment:
I would give this a more detailed name. What's the implication of the button being set to true? Gonna read the rest of the PR and come back

Reply:
This is used to decide if the 'Finish' button would be disabled or enabled. A.k.a whether a user can continue to next step. If a set of the copy button states are true, then use can continue to next page. Otherwise, user will not be able to continue.